.github/workflows/cleanup_model_artifacts.yml

```diff
@@ -0,0 +1,63 @@
+name: Cleanup Registered Models
```

Comment:
We need to create an issue about to sync this work with AzDO and reference it here.

Reply:
New issue created to track for azdo pipelines: https://github.com/microsoft/dstoolkit-mlops-v2/issues/174